.github/workflows/builds.yml

```diff
@@ -170,12 +170,12 @@ jobs:
         shell: bash
         env:
           GH_TOKEN: ${{ secrets.FERDIUM_PUBLISH_TOKEN }}
-          CSC_IDENTITY_AUTO_DISCOVERY: false
+          CSC_IDENTITY_AUTO_DISCOVERY: true
           # TODO: Commented out the code signing process for now (so as to at least get unsigned nightlies available for testing)
           # APPLEID: ${{ secrets.APPLEID }}
           # APPLEID_PASSWORD: ${{ secrets.APPLEID_PASSWORD }}
-          # CSC_LINK: ${{ secrets.CSC_LINK }}
-          # CSC_KEY_PASSWORD: ${{ secrets.CSC_KEY_PASSWORD }}
+          CSC_LINK: ${{ secrets.CSC_LINK }}
```

Comment:
If we enable these, are we now saying that the nightlies will be signed for the windows builds? If so, do we have the required certs, etc? If we dont, then these 2 lines (and the corresponding repeated ones) should remain commented out

Reply:
> If we enable these, are we now saying that the nightlies will be signed for the windows builds? If so, do we have the required certs, etc? If we dont, then these 2 lines (and the corresponding repeated ones) should remain commented out

I think this lines are referring to macOS build, right? We have to comment those on the windows build (changed on latest commits)